### ISSUE TITLE
Documentation: Note the type of files that Realm may create and what they do

### ISSUE THREAD

**Issue by diegomontoya** [realm/realm-java#3025]
I just noticed that realm is also creating ".management" files in-addition to .lock, .log, .note files. I don't think the files and their use are documented. This would would be a good addition to documentation.

Would also be good to note if each file is safe to remove if the realm is not open.

**Comment by cmelchior** [comment]
Yes, that sounds like something we should document. All auxiliary files are safe to delete if the Realm is no longer open. If you delete them while it is open you risk corrupting the file, sinc threads or processes can no longer see that others are working on the file.
